src/qt/transactionfilterproxy.cpp

```diff
@@ -51,8 +52,11 @@ bool TransactionFilterProxy::filterAcceptsRow(int sourceRow, const QModelIndex &
         return false;
     if(datetime < dateFrom || datetime > dateTo)
         return false;
-    if (!address.contains(searchString, Qt::CaseInsensitive) && !label.contains(searchString, Qt::CaseInsensitive))
+    if (!address.contains(searchString, Qt::CaseInsensitive)
+     && !  label.contains(searchString, Qt::CaseInsensitive)
```

Comment:
Nit, drop alignments?

Reply:
Why?

Reply:
As it is `clang-format` makes it one line. This is preserved by clang-format:
```cpp
    if (!address.contains(searchString, Qt::CaseInsensitive) &&
        !label.contains(searchString, Qt::CaseInsensitive) &&
        !txid.contains(searchString, Qt::CaseInsensitive)) {
```